### PR TITLE
Intellij 13

### PR DESCRIPTION
**Pull request by jtydhr88**
thanks @dmarks2 a lot for contributing the tests

### CODE REVIEW

**Comment by jtydhr88** [comment]
thanks @dmarks2 a lot for contributing the tests!

File: build.gradle [hunk 1]
```diff
@@ -93,6 +93,12 @@ task verifyBlade(type: Verify)
 
 check.dependsOn checkSourceFormatting
 
+test {
```
Comment (gamerson) [inline build.gradle]: @jtydhr88  Need to sort this clousre.

File: build.gradle [hunk 2]
```diff
@@ -102,6 +108,8 @@ dependencies {
 	compile group: "commons-configuration", name: "commons-configuration", version: "1.10"
 	compile group: "org.apache.ant", name: "ant", version: "1.10.1"
 	compile group: "org.dom4j", name: "dom4j", version: "2.1.0"
+
+	testCompile group: "junit", name: "junit", version: "4.11"
```
Comment (gamerson) [inline build.gradle]:
```suggestion
	testCompile group: "junit", name: "junit", version: "4.12"
```

File: build.gradle [hunk 3]
```diff
@@ -115,7 +123,9 @@ formatSource {
 }
 
 intellij {
-	plugins "gradle", "maven", "terminal", "JavaEE", "java-i18n", "properties", "CSS", "JavaScriptLanguage"
+	plugins "gradle", "maven", "terminal", "JavaEE", "java-i18n", "properties", "CSS", "JavaScriptLanguage",
```
Comment (gamerson) [inline build.gradle]: @jtydhr88  please sort.

File: src/main/java/com/liferay/ide/idea/language/LiferayXmlFileReferenceFilterPattern.java [hunk 4]
```diff
@@ -135,8 +134,10 @@ public boolean isClassAcceptable(Class hintClass) {
 					"portlet-decorator-thumbnail-path"));
 			put(
 				LiferayDefinitionsResourceProvider.XML_NAMESPACE_LIFERAY_LOOK_AND_FEEL_7_1_0,
-				_tagsMap.get(LiferayDefinitionsResourceProvider.XML_NAMESPACE_LIFERAY_LOOK_AND_FEEL_7_0_0));
-
+				Arrays.asList(
+					"root-path", "templates-path", "css-path", "images-path", "javascript-path",
```
Comment (gamerson) [inline src/main/java/com/liferay/ide/idea/language/LiferayXmlFileReferenceFilterPattern.java]: Do these entity names match the order in the DTD?  if not can you sort them accordingly?

**Comment by gamerson** [comment]
hey @dmarks2  Thanks so much for submitting these tests.  Sorry for the delay, I've been at NAS.

How do you invoke them from commandline or in Intellij?  Just `gradle test`?

**Comment by jtydhr88** [comment]
@gamerson just gradle test

File: src/main/java/com/liferay/ide/idea/language/LiferayXmlFileReferenceFilterPattern.java (hunk 4 above)
Comment (jtydhr88) [reply src/main/java/com/liferay/ide/idea/language/LiferayXmlFileReferenceFilterPattern.java]: it already match by the dtd

**Comment by dmarks2** [comment]
Hello @gamerson,
correct, you can run the tests using `gradle test`. You can also run individual tests directly in IntelliJ IDEA using right click --> Run.

Additionally the unit tests are run automatically during a normal `gradle build`. That means you can already see that the tests are executed and the tests are successful in the Jenkins builds (see here for example: https://jenkins-ide.wedeploy.io/job/liferay-intellij-plugin/43/testReport/)

**Pull request merged**